src/Explorer.js

```diff
@@ -495,11 +502,16 @@ class ArgView extends React.PureComponent<ArgViewProps, ArgViewState> {
         setArgValue={this._setArgValue}
         getDefaultScalarArgValue={this.props.getDefaultScalarArgValue}
         makeDefaultArg={this.props.makeDefaultArg}
+        onRunOperation={event => this.props.onRunOperation()}
```

Comment:
Create `this._handleRunOperation` so it doesn't create a new function on each re-render. Can help with performance if it's using PureComponent.